perllib/FixMyStreet/Script/Alerts.pm

```diff
@@ -39,6 +39,7 @@ sub send() {
                    $item_table.name as item_name, $item_table.anonymous as item_anonymous,
                    $item_table.confirmed as item_confirmed,
                    $item_table.photo as item_photo,
+                   $item_table.problem_state as item_problem_state,
```

Comment:
Should update the comment at line 22 (which is already out of date). Basically this assumes it's the comment table, doesn't it, while looking generic.

Reply:
Yes, it does.